docs/snippets/angular/button-story-decorator.ts.mdx

```diff
@@ -1,30 +1,25 @@
 ```ts
 // button.stories.ts
 
-import { Meta, Story, componentWrapperDecorator, moduleMetadata } from '@storybook/angular';
+import { componentWrapperDecorator, moduleMetadata } from '@storybook/angular';
 
-import { ButtonComponent } from './button.component';
+import { Button } from './button.component';
 import { ParentComponent } from './parent.component'; // ParentComponent contains ng-content
```

Comment:
Maybe `Parent` for consistency? You should probably use `Button` / `Parent` or `ButtonComponent` / `ParentComponent` everywhere for consistency

Reply:
Parent seems a good fit and Button as well for consistency. I'll update them in a bit

Reply:
I've addressed this with the latest commit. Marking it as solved.